extensions/amp-analytics/analytics-vars.md

```diff
@@ -0,0 +1,105 @@
+# <a name="amp-analytics"></a> Variables supported in `amp-analytics`
+
```

Comment:
Structural suggestion that may require a conversation to explain a bit better: You don't need the two h2 levels. I would suggest removing 'using' section and making this at the top level, just as the intro the doc, and then move each of the supported variables up to be h2s. 

This does two things: it introduces how the variables are used, briefly, and also raises up the actual supported variables to h2 level, making the document much easier to scan. The actual variable values are mainly what developers will look for, so makes sense to give them more weight.